recipe/meta.yaml

```diff
@@ -1,31 +1,33 @@
 {% set name = "mono" %}
-{% set version = "5.14.0.177" %}
-{% set sha256 = "d4f5fa2e8188d66fbc8054f4145711e45c1faa6d070e63600efab93d1d189498" %}
+{% set version = "6.12.0.90" %}
+{% set sha256 = "51de5c02ad511333f93ff585bca54c8784de35af4ff27b759d16b46c4402cdac" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.bz2
-  url: http://download.mono-project.com/sources/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  url: https://download.mono-project.com/sources/mono/mono-{{ version }}.tar.xz
   sha256: {{ sha256 }}
 
 build:
-  number: 1001
-  skip: True  # [win]
+  number: 0
+  skip: true  # [win]
 
 requirements:
   build:
     - perl
-    - automake
+    - automake >=1.16
     - libtool
     - pkg-config
     - cmake
+    - autoconf
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-  host:
     - python
+    - make       # [unix]
+  host:
+    - zlib
     - gettext
```

Comment:
Why do you need `gettext` in host?

Reply:
As far as I can see, `gettext` is in host already (it's not added in this PR). `gettext` is used quite a bit throughout mono (see https://github.com/mono/mono/search?q=gettext).

It would be great to see this PR merged :)

Reply:
Not really. That's a different thing.